panel/models/reactive_html.ts

```diff
@@ -138,7 +138,7 @@ export class ReactiveHTMLView extends HTMLBoxView {
     super.connect_signals()
     this.connect(this.model.properties.children.change, async () => {
       this.html = htmlDecode(this.model.html) || this.model.html
-      await this.rebuild()
+      await this.update_children()
```

Comment:
Can you explain this? It's breaking things.

Reply:
I see, `rebuild` was removed.

Reply:
I didn't realize panel was using this API, which was used in bokehjs only once and was duplicating work unnecessarily. That's why we need to test panel continuously (at least whether it builds) in bokeh's CI.